src/lib/isEmpty.js

```diff
@@ -1,12 +1,12 @@
 export const isEmpty = (obj) => {
 	if (obj === null || obj === undefined) {
-		return True;
+		return true;
 	} else if (Array.isArray(obj) || typeof obj === "string") {
 		return obj.length === 0;
 	} else if (
 		["boolean", "number", "bigint", "symbol", "function"].includes(typeof obj)
 	) {
-		throw new Error(f`Unsuported type: ${typeof obj}`);
+		throw new Error(`Unsuported type: ${typeof obj}`);
 	} else {
 		return Object.keys(obj).length === 0;
 	}
```

Comment:
Oh dear, I am such a python programmer.....

Reply:
Oh, and I missed a typo!

Reply:
Crap, I'll put up another PR!